docs/src/user-guide/command-line-interface.md

```diff
@@ -233,11 +233,11 @@ echo '3 ** 4' | npx eslint --stdin --parser-options ecmaVersion:7 # succeeds, ya
 
 Changes the folder where plugins are resolved from.
```

Comment:
For consistency, we should probably just stick with "directory" instead of bouncing back and forth with "folder"

```suggestion
Changes the directory where plugins are resolved from.
```